coffee_cmd/src/main.rs

```diff
@@ -78,6 +78,19 @@ async fn main() -> Result<(), CoffeeError> {
             }
             Err(err) => Err(err),
         },
+        CoffeeCommand::Search { plugin } => match coffee.search(&plugin).await {
+            Ok(val) => {
+                let val = val["search"].as_str().unwrap();
+                if val == "not found" {
+                    term::error(&("unable to locate plugin ".to_owned() + &plugin));
```

Comment:
this part of the if needs to be removed! The error is returned out of the match and then print on the terminal at the line 97

Reply:
Yes thanks for pointing that out